toolchain/get_app_ver.py

```diff
@@ -0,0 +1,18 @@
+import urllib.request
+import json
+import os
+
+store_api = urllib.request.urlopen("https://itunes.apple.com/jp/lookup?id=1016318735")
+store_b = store_api.read()
+store_j = store_b.decode('utf-8')
+store_d = json.loads(store_j)
+
+results_l = store_d.get("results")
+results_j = json.dumps(results_l[0])
+results_d = json.loads(results_j)
+
+app_ver = results_d['version']
+
+print("get app_ver:", app_ver)
+os.environ["VC_APP_VER"] = app_ver
```

Comment:
Changes to environ aren't visible outside the process so the script currently doesn't do anything.

I think the best way would be to print just the app version, then it could be easily used in a shell script, like

    VC_APP_VER=$(python3 toolchain/get_app_ver.py) python3 app.py